---
title: Existing data
keywords:
summary: Before investing time, effort and resources into generating new data, look into what has already been done in the past in your field. By reusing data you increase its value and minimize redundancy.
sidebar: mydoc_sidebar
permalink: existing_data.html
---

## Sources of existing data

**Bibliographic research**<br/>Start with a bibliographic research. If you find an interesting publication without any available data, you can contact the authors and request access to their data. If their data are not available or you didn’t find any interesting publication, you can look for existing data in many repositories.

**Data repositories**<br/>Repositories or databases can also contain data that are not linked to any manuscript or “results paper”.
Repositories can be general, data type specific or discipline specific.

## ELIXIR Deposition Databases for Biomolecular Data
ELIXIR recommends the following [databases](https://elixir-europe.org/platforms/data/elixir-deposition-databases) for specific data type
 * Functional genomics: [ArrayExpress](https://www.ebi.ac.uk/arrayexpress/)
 * Computational models of biological processes: [BioModels](https://www.ebi.ac.uk/biomodels/)
 * Descriptions and metadata about biological samples used in research: [BioSamples](https://www.ebi.ac.uk/biosamples/)
 * Descriptions of biological studies: [BioStudies](https://www.ebi.ac.uk/biostudies/)
 * Personally identifiable genetic and phenotypic data resulting from biomedical research projects: [EGA](https://ega-archive.org)
 * Electron microscopy density maps of macromolecular complexes and subcellular structures: [EMBD](https://www.ebi.ac.uk/pdbe/emdb/)
 * Nucleotide sequence information: [ENA](https://www.ebi.ac.uk/ena)
 * Genetic variation data from all species: [EVA](https://www.ebi.ac.uk/eva/)
 * Molecular interaction data: [IntAct](https://www.ebi.ac.uk/intact/)
 * Metabolomics experiments and derived information: [MetaboLights](https://www.ebi.ac.uk/metabolights/)
 * Biological macromolecular structures: [PDBe](https://www.ebi.ac.uk/pdbe/)
 * Proteomics experiments and derived information: [PRIDE](https://www.ebi.ac.uk/pride/archive)

## Other lists of recommended repositories
Scientific journals and communities have compiled a number of lists of recommended repositories, searchable by discipline and other characteristics

* Nature - Scientific Data: [Recommended Data Repositories](https://www.nature.com/sdata/policies/repositories#general)
* Repositories used by Flemish research infrastructures: https://mronocoroni.shinyapps.io/20200325/
* FAIRsharing: [Catalogue of databases](https://fairsharing.org/databases/)
* OpenAIRE: [International search engines for academic and scientific research](https://explore.openaire.eu)
* DataCite: [Registry of Research Data Repositories]( https://www.re3data.org)

## How to cite an existing dataset
**Complete citation**<br/>Author(s), Year, Dataset Title, Identifier, Repository, Version.

**Short citation**<br/>Identifier, Version (if applicable).

**Identifiers** are machine readable alphanumeric strings provided by repositories. Identifiers can be:
* Accession number<br/>example: E-MTAB-NNNN
* DOIs<br/>example: doi: 10.1038/d41586-018-03071-1
