---
title: UniProt
keywords:
summary: Empty summary
sidebar: mydoc_sidebar
permalink: uniprot.html
---
## UniProt features
**How to submit data**\\
[UniProt submission guidelines](https://www.uniprot.org/help/submissions).
[Submit protein sequences](https://www.ebi.ac.uk/swissprot/Submissions/spin/help).

You can submit full or partial protein sequences, sequences determined at protein level and associated functional annotation to UniProtKB.
* Edman degradation or manual interpretation of MS/MS spectra data is submitted via [SPIN](https://www.ebi.ac.uk/swissprot/Submissions/spin/account/login).
* Translated nucleotide sequences should be submitted to [ENA](https://www.ebi.ac.uk/ena/browser/home).
* MS/MS data should be submitted to [PRIDE](https://www.ebi.ac.uk/pride/).


**Non-Personal vs Personal data**\\
Non-personal data (non-identifiable).

**Access to data**\\
Open access.

**Embargo**\\
Possible. There are three options for the release date:
* Data may be published without further notice.
* Data must be kept confidential until publication.
*Data may be made public after the specified date.

**Data licence**\\
UniProt applies the Creative Commons Attribution (CC BY 4.0) License to all copyrightable parts of its databases. It cannot provide unrestricted permission regarding the use of the data, as some data may be covered by patents or other rights ([UniProt License & disclaimer](https://www.uniprot.org/help/license)).

**Data/Experiments types**
* Edman degradation.
* Manual interpretation of MS/MS data.
* Mascot or similar search algorithms.
* Translation of a nucleotide sequence.

**Metadata**\\
Required:
* *Protein name*.
* *Sequencing method*.
* *Organism*.
* *Sequence*.
* *Citations*.
* *Confidentiality (see Embargo, above)*.

Optional:
* Properties of the protein: *Mass spectrometry; Function; Tissue specificity; Similarity; EC number; Catalytic activity; Cofactor; Pathway; Enzyme regulation; Vmax; KM data; Quarternary structure ; Allergenicity; Subcellular location; Posttranslational modification; Induction; Developmental stage; Optimum temperature; Optimum pH; Redox potential; Absorption; 2D-PAGE results; Miscellaneous*
* Sequence features of the protein: *Uncertainty in the sequence; Post-translationally modified residue; Disulfide bond; Active site; Glycosylation site; Metal ion-binding site; Nucleotide-binding region; DNA-binding region; Other binding site; Transmembrane region; Other domain of interest; Other site of interest; Sequence variation*.

A complete description of the metadata requirements can be found at the [SPIN submission guidelines](https://www.ebi.ac.uk/swissprot/Submissions/spin/help#publish).

**Ontology**\\
UniProt has its own [rdf schema ontology](http://ontologies.berkeleybop.org/mi.owl).

**Data documentation**\\
UniProt doesn’t allow upload of README file. All relevant information about the data (metadata) need to be provided in the designated fields. See “Metadata” section above.

**File format(s)**\\
UniProt submission system (SPIN) takes single-letter amino acid code. A complete description of the UniProtKB/Swiss-Prot format is given [here](https://web.expasy.org/docs/userman.html#genstruc).

If you are familiar with the UniProtKB/Swiss-Prot format you are encouraged to submit your updates and/or corrections using that format, but this is not a prerequisite and it is preferred that you provide a longer textual description of what you are submitting rather than spend time trying to fit your data to the UniProt format.

**Data volume and costs**\\
No limit for data volume. No costs.

**Data quality**\\
UniProtKB/Swiss-Prot is a curated database and not an unedited archive or repository. It makes no promise as to whether it will accept to represent in a given entry all data sent to us.

**Identifiers**\\
An accession number is minted once the submission has been processed by a curator.

## Guide and manual
https://www.uniprot.org/docs/userman.htm


## Tips for data submission
